workflow_scripts/ort_test_dir_utils.py

```diff
@@ -209,7 +214,12 @@ def run_test_dir(model_or_dir):
     if not test_dirs:
         raise ValueError("No directories with name starting with 'test' were found in {}.".format(model_dir))
 
-    sess = ort.InferenceSession(model_path)
+    # Start from ORT 1.10, ORT requires explicitly setting the providers parameter if you want to use execution providers
+    # other than the default CPU provider (as opposed to the previous behavior of providers getting set/registered by default
+    # based on the build flags) when instantiating InferenceSession.
+    # Following code assumes NVIDIA GPU is available, you can specify other execution providers or don't include providers parameter
+    # to use default CPU provider.
+    sess = ort.InferenceSession(model_path, providers=['CUDAExecutionProvider'])
```

Comment:
Has ORT decided which the default provider (CPU or GPU) is for InferenceSession for these utilities (ort_test_dir_utils.py) yet? I didn't see [related changes](https://github.com/microsoft/onnxruntime/blob/master/tools/python/ort_test_dir_utils.py) under ORT repo for now. Ideally I think the utilities from ORT here in onnx/models should be align with ORT if possible.

Although I don't have the context here, it makes more sense to me that general ORT package should use InferenceSession with default CPU provider and GPU package (onnxruntime-gpu) should use InferenceSession with default GPU provider. Am I correct? And please note that now CI in onnx/models only verifies uploaded models with general ORT package (onnxruntime).

Reply:
I think it will bring another topic:
What providers should be tested for those uploaded ONNX models in ONNX model zoo? Only CPU, only GPU or both?